docs/operate/get-started/setup-micro/_index.md

```diff
@@ -0,0 +1,404 @@
+---
+linkTitle: "Set up an ESP32"
+title: "Set up an ESP32"
+weight: 25
+layout: "docs"
+type: "docs"
+images: ["/installation/thumbnails/install.png"]
+imageAlt: "Install Viam"
+no_list: false
+description: "Install the lightweight version of the software that drives hardware and connects your device to the cloud."
+prev: "/operate/get-started/basics/"
+next: "/operate/get-started/other-hardware/micro-module/"
+aliases:
+  - /installation/microcontrollers/
+  - /installation/prepare/microcontrollers/
+  - /installation/prepare/microcontrollers/
+  - /build/micro-rdk/
+  - /get-started/installation/microcontrollers/
+  - /installation/viam-micro-server-setup/
+  - /operate/reference/viam-micro-server/viam-micro-server-troubleshooting/
+---
+
+Viam maintains a [lightweight version of `viam-server`](/operate/reference/viam-micro-server/) for microcontrollers.
+
+## Supported microcontrollers
+
+ESP32 microcontrollers must have at least 2 cores, 384kB SRAM, 2MB PSRAM and 8MB flash to work with Viam.
+The following microcontrollers have been tested with Viam:
+
+- [ESP32-WROVER Series](https://www.espressif.com/en/products/modules/esp32)
+
+## About ESP32 microcontroller setup
+
+Because microcontrollers do not run operating systems and are instead flashed with firmware, the setup process is different than for regular computers and SBCs.
```

Comment:
```suggestion
SBCs, laptops, and desktops run a complete operating system.
Microcontrollers run dedicated firmware that runs independently of an operating system.
Because microcontrollers do not run an operating system, they use a different setup process than SBCs, laptops, and desktops.
```

[opt] This sentence is doing a lot of work. I took a stab at breaking up the 'operating system vs no OS', 'SBCs/computers vs. microcontrollers', and 'different setup process' ideas up into separate, more readable sentences. I don't love front-loading a sentence that isn't about microcontrollers, but maybe this will give you a better idea.

Reply:
Smaller change, slightly shorter and keeps the emphasis on microcontrollers. Thoughts?
```suggestion
Microcontrollers do not run operating systems and are instead flashed with dedicated firmware.
Because of this, the microcontroller setup process is different than for SBCs, laptops, and desktops.
```

Reply:
ship it! looks even better than my suggestion.